test/other-issues/lazy-relations-loading-via-base-entitiy-finders/index.ts

```diff
@@ -0,0 +1,33 @@
+import {Post} from "./entity/Post";
+import {Category} from "./entity/Category";
+import "reflect-metadata";
+import {closeTestingConnections, createTestingConnections, reloadTestingDatabases} from "../../utils/test-utils";
+import {Connection} from "../../../src/connection/Connection";
+import {expect} from "chai";
+import {PromiseUtils} from "../../../src";
+
+describe("lazy-relations-loading-via-base-entity-finders", () => {
```

Comment:
Can this love in the standard functional tests instead of in this part of the tests

Reply:
I am not sure what do you mean here. Can you explain?